src/components/visualizations/chart/test/colorFactory.spec.ts

```diff
@@ -0,0 +1,270 @@
+import {
+    TreeMapColorStrategy,
+    MeasureColorStrategy,
+    AttributeColorStrategy,
+    HeatMapColorStrategy,
+    ColorFactory,
+    IColorStrategy
+} from '../colorFactory';
+
+import { DEFAULT_COLOR_PALETTE, HEATMAP_BLUE_COLOR_PALETTE } from '../../utils/color';
+
+import * as fixtures from '../../../../../stories/test_data/fixtures';
+
+import {
+    VIEW_BY_DIMENSION_INDEX,
+    STACK_BY_DIMENSION_INDEX
+} from '../constants';
+
+import {
+    findMeasureGroupInDimensions,
+    findAttributeInDimension
+} from '../chartOptionsBuilder';
+
+function getMVS(dataSet: any) {
```

Comment:
this is the same function like in chartOptionsBuilder.spec file. Move it to the helper and import in both test files